### PR TITLE
docs(@angular-devkit/schematics-cli): Add README

### PR DESCRIPTION
**Pull request by kyliau**
This commit adds a README for the package.

### CODE REVIEW

File: packages/angular_devkit/schematics_cli/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Schematics CLI
+
+This package contains the executable for running a [Schematic](https://www.npmjs.com/package/@angular-devkit/schematics).
```
Comment (alan-agius4) [inline packages/angular_devkit/schematics_cli/README.md:3]: Should this link to the NPM package or readme within Github https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/schematics?

Comment (kyliau) [reply packages/angular_devkit/schematics_cli/README.md]: I think GitHub is better, the link has been updated.

Comment (alan-agius4) [reply packages/angular_devkit/schematics_cli/README.md]: Maybe relative is better?

**Review by alan-agius4 — APPROVED** [review]

**Review by alan-agius4 — APPROVED** [review]

File: packages/angular_devkit/schematics_cli/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Schematics CLI
+
+This package contains the executable for running a [Schematic](/packages/angular_devkit/schematics).
```
Comment (alan-agius4) [inline packages/angular_devkit/schematics_cli/README.md:3]: Nit: 
```suggestion
This package contains the executable for running a [Schematic](/packages/angular_devkit/schematics/README.md).
```

**Review by alan-agius4 — APPROVED** [review]

**Review by clydin — APPROVED** [review]

**Pull request merged**